entity-framework/core/managing-schemas/migrations/applying.md

```diff
@@ -343,3 +343,23 @@ Note that `Migrate()` builds on top of the `IMigrator` service, which can be use
 >
 > * Carefully consider before using this approach in production. Experience has shown that the simplicity of this deployment strategy is outweighed by the issues it creates. Consider generating SQL scripts from migrations instead.
 > * Don't call `EnsureCreated()` before `Migrate()`. `EnsureCreated()` bypasses Migrations to create the schema, which causes `Migrate()` to fail.
+
+## Migration locking
```

Comment:
is this the right place to put the extra info? @AndriySvyryd

Reply:
@maumar do we think users need to know about the migration lock, and the implementation difference between SQLite and SQL Server? I do think it makes sense to have a "what's new" section on it (because it's a new feature), but I'm not sure why a user using migrations should need to know that the migration lock exists or how it works...

In contrast, further up on this page there are warnings against using Migrate() because of concurrency on startup - that's exactly what the migration lock fixes, so we should look at amending the page to not scare people away from Migrate() (when using 9.0 and above).

Reply:
sounds good, I removed the details section and added a short note instead

Reply:
ended up removing most of the superfluous info, but left some around. Added a documentation note on what to do when lock doesn't get lifted on sqlite (due to failed transaction). User needs to manually drop the migration lock table, so little info adds some context why user needs to do this